checker/src/org/checkerframework/checker/lock/LockChecker.java

```diff
@@ -8,6 +12,12 @@
 /**
  * @checker_framework.manual #lock-checker Lock Checker
  */
-@TypeQualifiers({ LockHeld.class, LockPossiblyHeld.class })
+// The Lock Checker uses two distinct type qualifier hierarchies:
+// the @LockPossiblyHeld and the @GuardedByInaccessible hierarchies.
+@TypeQualifiers({ LockPossiblyHeld.class, LockHeld.class,
+                  GuardedByInaccessible.class, GuardedBy.class,
+                  javax.annotation.concurrent.GuardedBy.class,
```

Comment:
Why are the javax and jcip annotations supported? They are just aliases and don't need to be part of the hierarchy.